src/coreclr/src/System.Private.CoreLib/ILLinker/ILLink.Suppressions.cs

```diff
@@ -0,0 +1,276 @@
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetEvents")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetEvents(System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetField(System.String,System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetFields(System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetMember(System.String,System.Reflection.MemberTypes,System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetMembers(System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetMethodImpl(System.String,System.Reflection.BindingFlags,System.Reflection.Binder,System.Reflection.CallingConventions,System.Type[],System.Reflection.ParameterModifier[])")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetMethods(System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetNestedType(System.String,System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetNestedTypes(System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetProperties(System.Reflection.BindingFlags)")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.GetPropertyImpl(System.String,System.Reflection.BindingFlags,System.Reflection.Binder,System.Type,System.Type[],System.Reflection.ParameterModifier[])")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.SymbolType.InvokeMember(System.String,System.Reflection.BindingFlags,System.Reflection.Binder,System.Object,System.Object[],System.Reflection.ParameterModifier[],System.Globalization.CultureInfo,System.String[])")]
+[assembly: UnconditionalSuppressMessage ("ILLinker", "IL2047", Scope = "member", Target = "M:System.Reflection.Emit.TypeBuilder.GetConstructorImpl(System.Reflection.BindingFlags,System.Reflection.Binder,System.Reflection.CallingConventions,System.Type[],System.Reflection.ParameterModifier[])")]
```

Comment:
Above you seem to disable this warning all up - we should either disable it, or add the baselines for it... not both.